### DISCUSSION TITLE
Notifications not clearing in-app on app re-open

### DISCUSSION THREAD

**Discussion by ghost** [community/community#65534] [Mobile]
In the latest GitHub iOS update, swiping to remove notifications appears to work, however when switching away from the app and back in again, the notifications are all there unread again as if they were never removed.

In the below images, you'll all notifications are unread again after clearing them.

Switching to "Done", all those notifications then also appear there also.

Switching back to unread notifications again, they're all then gone again as I would have expected.

![image](https://github.com/community/community/assets/95468948/31aa2c5d-c5dc-43a1-9596-b9cd45e33520)

![image](https://github.com/community/community/assets/95468948/bdccdce5-d3d0-484e-a906-ff4901c2dcec)

![image](https://github.com/community/community/assets/95468948/e1601b6f-768c-450b-8254-f7032c741609)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.128.0 (108490092)
Device: iPhone 13 Pro (iOS 17.0)
</details>

**Comment by trixieboyle** [comment]
a03d9c3cd6510f6a04435ebac1801e86113a4b71

**Comment by trixieboyle** [comment]
> a03d9c3cd6510f6a04435ebac1801e86113a4b71

TammyBoyle

**Comment by trixieboyle** [comment]
> > a03d9c3cd6510f6a04435ebac1801e86113a4b71
> 
> TammyBoyle

**Comment by trixieboyle** [comment]
> > > a03d9c3cd6510f6a04435ebac1801e86113a4b71
> > 
> > TammyBoyle 
> 
>

**Comment by trixieboyle** [comment]
> a03d9c3cd6510f6a04435ebac1801e86113a4b71
